domenetjenester/uttak/src/main/java/no/nav/foreldrepenger/domene/uttak/fakta/omsorg/AvklarLøpendeOmsorgAksjonspunktUtleder.java

```diff
@@ -46,12 +45,19 @@ public Optional<AksjonspunktDefinisjon> utledAksjonspunktFor(UttakInput input) {
             return Optional.empty();
         }
         // Trenger ikke rebekrefte
-        if (BehandlingType.REVURDERING.equals(ref.behandlingType())
-            && ytelsesFordelingRepository.hentAggregatHvisEksisterer(ref.behandlingId()).filter(a -> Boolean.TRUE.equals(a.getOverstyrtOmsorg())).isPresent()) {
-            return Optional.empty();
+        var skjæringstidspunkt = input.getSkjæringstidspunkt().orElseThrow();
+        if (BehandlingType.REVURDERING.equals(ref.behandlingType())) {
+            var ytelseFordelingAggregat = ytelsesFordelingRepository.hentAggregatHvisEksisterer(ref.behandlingId());
+            if (ytelseFordelingAggregat.filter(a -> Boolean.TRUE.equals(a.getOverstyrtOmsorg())).isPresent()) {
+                return Optional.empty();
+            }
+            if (ytelseFordelingAggregat.filter(a -> Boolean.FALSE.equals(a.getOverstyrtOmsorg())).isPresent()
```

Comment:
Ser riktig ut det. Synes fremdeles at overstyrtOmsorg-feltet i YF er litt forvirrende. For her settes det i AP både når det vurderes av saksbehandler og overstyrer?